mne/gui/_coreg_gui.py

```diff
@@ -1348,6 +1352,10 @@ def _get_hsp_visible(self):
         return self.hsp_always_visible or self.lock_fiducials
 
     @cached_property
+    def _get_eeg_as_hsp(self):
+        return self.use_eeg_as_hsp
```

Comment:
This is not necessary. These `_get_x` methods are for properties that are defined with `depends_on`, i.e. if you define a property that depends on others you use these functions to tell Traits how they're generated.